ci-operator/step-registry/servicemesh/kiali/cypress-tests/execute/servicemesh-kiali-cypress-tests-execute-commands.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+CONSOLE_URL=$(cat $SHARED_DIR/console.url)
+OCP_API_URL="https://api.${CONSOLE_URL#"https://console-openshift-console.apps."}:6443"
+OCP_CRED_USR="kubeadmin"
+OCP_CRED_PSW="$(cat ${SHARED_DIR}/kubeadmin-password)"
+
+oc login ${OCP_API_URL} --username=${OCP_CRED_USR} --password=${OCP_CRED_PSW} --insecure-skip-tls-verify=true
+hack/istio/install-testing-demos.sh -c oc -in ${SMCP_NAMESPACE}
+sleep 120
```

Comment:
If you are waiting for something to be provisioned, could we write some code to just check for what you are waiting for every 30ish seconds to see if it is ready? This may help in instances where 120 seconds isn't enough time.

Reply:
This is a bit more complicated than just waiting for some pod to be ready. Pods are starting/restarting during that process and we also have to wait for at least 1 minute to get some traffic history. I made some attempts in our automation to improve that but so far the sleep has been the most stable solution.

Reply:
That works for me, then!